### PR TITLE
Added index.ts file to helpers directory, and updated all imports, rearranged the docs

### PR DESCRIPTION
**Pull request by Metachaser24**
I added the index.ts file to the helpers directory.
- Updated all of the imports 
- Moved main README to packages/examples/ReadME
- Created a general readme at for the repo
- Added more in-depth zkEmailOverview Docs inside of docs folder

### CODE REVIEW

File: packages/helpers/index.ts [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+export * from './src/merkle';
+export * from './src/poseidonHash';
+export * from './src/rsa';
+export * from './src/shaHash';
+export * from './src/sshFormat';
+export * from './src/uncompress';
+export * from './src/binaryFormat';
+export * from './src/dkim/index';
+export * from './src/zkp';
+
```
Comment (Divide-By-0) [inline packages/helpers/index.ts:15]: remove empty lines at the end of file. we should have a linter hook automatically do this for you

File: packages/examples/twitter-verifier/twitter-verifier-circuits/helpers/generate-inputs.ts [hunk 2]
```diff
@@ -1,5 +1,10 @@
-import { bytesToBigInt, fromHex } from "@zk-email/helpers/src/binaryFormat";
-import { generateCircuitInputs } from "@zk-email/helpers/src/input-helpers";
+<<<<<<<< HEAD:packages/examples/twitter-verifier-example-docs/twitter-verifier-circuits/helpers/generate-inputs.ts
```
Comment (Divide-By-0) [inline packages/examples/twitter-verifier/twitter-verifier-circuits/helpers/generate-inputs.ts:1]: There's a diff left here

File: packages/examples/twitter-verifier/twitter-verifier-app/scripts/generate_input.ts [hunk 3]
```diff
@@ -12,12 +12,16 @@ import {
...
+<<<<<<<< HEAD:packages/examples/twitter-verifier-example-docs/twitter-verifier-app/scripts/generate_input.ts
+  int64toBytes, shaHash, partialSha, sha256Pad, CIRCOM_FIELD_MODULUS, MAX_HEADER_PADDED_BYTES, MAX_BODY_PADDED_BYTES, STRING_PRESELECTOR, dkimVerify
+} from "@zkemail/helpers";
+
 // @ts-ignore
-import { dkimVerify } from "@zk-email/helpers/src/dkim";
+========
+  int64toBytes, CIRCOM_FIELD_MODULUS, MAX_HEADER_PADDED_BYTES, MAX_BODY_PADDED_BYTES, STRING_PRESELECTOR, shaHash, partialSha, sha256Pad, dkimVerify
+} from "@zk-email/helpers";
+>>>>>>>> 85239ac2c41cd19a9dbc1b59145849fe2ddcd73f:packages/examples/twitter-verifier/twitter-verifier-app/scripts/generate_input.ts
```
Comment (Divide-By-0) [inline packages/examples/twitter-verifier/twitter-verifier-app/scripts/generate_input.ts:23]: Also a diff here

File: packages/examples/twitter-verifier/twitter-verifier-app/pages/MainPage.tsx [hunk 4]
```diff
@@ -8,16 +8,23 @@ import {
   generate_inputs,
   CircuitType,
 } from "../scripts/generate_input";
-import { rawEmailToBuffer } from "@zk-email/helpers/src/input-helpers";
-import { DKIMVerificationResult, verifyDKIMSignature } from "@zk-email/helpers/src/dkim";
+<<<<<<<< HEAD:packages/examples/twitter-verifier-example-docs/twitter-verifier-app/pages/MainPage.tsx
```
Comment (Divide-By-0) [inline packages/examples/twitter-verifier/twitter-verifier-app/pages/MainPage.tsx:11]: random git diffs here

File: packages/examples/twitter-verifier/README.md [hunk 5]
```diff
@@ -0,0 +1,419 @@
+# ZK Email Verify
```
Comment (Divide-By-0) [inline packages/examples/twitter-verifier/README.md:1]: There's a lot of info in here that applies to the broader codebase, not Twitter - can you isolate that out?

**Comment by Divide-By-0** [comment]
I want to merge this ASAP -- if moving the example repo like this causes problems, can you isolate your docs only changes for now and we can shift the other stuff later?

**Comment by Divide-By-0** [comment]
also i cant stack prs if you make changes from metachase24/docs_editor -- can you change your remote so this is a branch in this repo itself?

**Comment by Metachaser24** [comment]
Yes, i created a new branch and added only docs changes. I need permission to push my branch to the main repo

**Pull request closed without merging**